cognitedata/reveal
# feat(react-components): Add segmented buttons to the architecture

#### Type of change
![Feat](https://img.shields.io/badge/Type-Feat-green)  

## Description :pencil:

![image](https://github.com/user-attachments/assets/2129b706-b5d9-46bf-927c-e142548c8ba6)

You can make segmented button if deriver from `BaseOptionCommand `with option `isOptionMenu = false` the constructor

Done it for the new class `SetOrbitOrFirstPersonModeCommand`, with according button in `RevealButtons`.

## How has this been tested? :mag:

1. Run storybook
2. Open architecture
3. See toolbar  at left top.


**Know issue:** Can someone help to get the top toolbar in center. Tried but can't find the solution.
